gradle.properties

```diff
@@ -0,0 +1 @@
+version=0.1.1-SNAPSHOT
```

Comment:
What is the idea of this properties file? to be manually edited? How and when this is going to be changed?

Reply:
The plugin keeps track of this file, that's how it knows the current/new version